xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecStarfish.cpp

```diff
@@ -274,7 +274,7 @@ bool CDVDVideoCodecStarfish::AddData(const DemuxPacket& packet)
   auto pts = std::chrono::duration_cast<std::chrono::nanoseconds>(
       std::chrono::duration<double, std::ratio<1, DVD_TIME_BASE>>(packet.pts));
   auto dts = std::chrono::duration_cast<std::chrono::nanoseconds>(
-      std::chrono::duration<double, std::ratio<1, DVD_TIME_BASE>>(packet.pts));
+      std::chrono::duration<double, std::ratio<1, DVD_TIME_BASE>>(packet.dts));
```

Comment:
Seems really a typo was coming in on https://github.com/xbmc/xbmc/commit/b0b5835b9f4f143964a18e2350ce1c3ea866f662#diff-8fe0b7338f2ba59706bcb7a6d056687f928f954b048e01a0e37130df06a1ef38R256-R259 by https://github.com/xbmc/xbmc/pull/23137

Ping @wsnipex

Reply:
Yes, it's a regression introduced in the cleanup